cluster/src/test/java/com/hartwig/pipeline/testsupport/BucketInputOutput.java

```diff
@@ -0,0 +1,29 @@
+package com.hartwig.pipeline.testsupport;
+
+import static java.lang.String.format;
+
+public class BucketInputOutput {
```

Comment:
Find this abstraction confusing. push/pull sounds like a kind of stack-like thing? but then resource doesn't quite fit :) I think perhaps calling push "output", pull "input" would be better.

Reply:
I think you're right. I actually meant to change it before I pushed. I'll fix that up.